modules/aws/core/globals/main.tf

```diff
@@ -1,5 +1,6 @@
 locals {
   sonar_tarball_s3_key_map = {
+    "4.13"      = "jsonar-4.13.0.10.0-rc7_20230817193838.tar.gz"
```

Comment:
You don't need this since you are going to use the tarball_location variable which bypasses this map. In addition, I think it is better if we don't commit code with RC tarballs/AMIs, even if it doesn't eventually reach master.